CI/physmon/workflows/physmon_ckf_tracking.py

```diff
@@ -105,7 +105,7 @@ def run_ckf_tracking(truthSmearedSeeded, truthEstimatedSeeded, label):
                 minPt=500 * u.MeV,
                 impactMax=3 * u.mm,
             ),
-            SeedFinderOptionsArg(bFieldInZ=1.99724 * u.T, beamPos=(0.0, 0.0)),
+            SeedFinderOptionsArg(bFieldInZ=2 * u.T),
```

Comment:
Do we know, why there was that very specific field of `1.99724 T` before?

Reply:
before https://github.com/acts-project/acts/pull/2132 this basically meant `2 T` but after that change not anymore